parm/config/gfs/config.base

```diff
@@ -448,6 +448,15 @@ export INCVARS_EFOLD="5"
 export netcdf_diag=".true."
 export binary_diag=".false."
 
+# Cellular automata
+case ${RUN} in
```

Comment:
I think this is leftover from mixed `DO_CA` settings, but is the `case` section necessary now?